tests/assertions/ocp/ocp4-cis-4.14.yml

```diff
@@ -0,0 +1,355 @@
+rule_results:
+  e2e-cis-accounts-restrict-service-account-tokens:
```

Comment:
One potential way to simplify this would be to use:

```
e2e-cis-accounts-restrict-service-account-tokens: MANUAL
e2e-cis-api-server-admission-control-plugin-namespacelifecycle: PASS
e2e-cis-api-server-encryption-provider-cipher: FAIL
```

Then just have the file name be `tests/assertions/ocp/ocp4-cis-4.14-default-results.yml`

Then for the remediated results we could do the same thing:

```
e2e-cis-accounts-restrict-service-account-tokens: MANUAL
e2e-cis-api-server-admission-control-plugin-namespacelifecycle: PASS
e2e-cis-api-server-encryption-provider-cipher: PASS
```

But have them in a separate file called `tests/assertions/ocp/ocp4-cis-4.14-remediated-results.yml`

This is just an idea, and we can continue iterating on it in subsequent patches if we decide to do it, but it would simplify the overall file structure, and the look ups we need to perform in the suite.

Reply:
I don't have a strong opinion on the file structure, both seem good to me but I slightly lean on the format proposed by Vincent.

Reply:
That makes sense - let's reuse what we have.